nix/fetchNixpkgs.nix

```diff
@@ -1,6 +1,6 @@
 { rev                             # The Git revision of nixpkgs to fetch
 , sha256                          # The SHA256 of the downloaded data
-, outputSha256 ? null             # The SHA256 output hash
+, outputSha256 ? null             # The SHA256 fixed-output hash
```

Comment:
I'm not sure what the difference between `sha256` and `outputSha256` is

Reply:
`sha256` is the fixed-output hash for the intermediate tarball derivation in the Nix < 1.12 branch and `outputSha256` is the fixed-output hash for the final `nixpkgs` derivation in the Nix < 1.12 branch.

In the Nix >= 1.12 branch `sha256` is unused because we only need the _final_ fixed-output derivation hash, which is `outputSha256`.